src/app.js

```diff
@@ -0,0 +1,24 @@
+require('env2')('./config.env');
+
+const { join } = require('path');
+
+const express = require('express');
```

Comment:
these are related to this pull request #20, so please remove the duplicate changes to avoid conflicts. Turn this pull request into a draft and whenever #20 is merged you can set this one ready to review. By the way I think all approved so you can merge it now.

Reply:
ok I'm working on that